pkg/acsengine/const.go

```diff
@@ -80,6 +80,50 @@ const (
 
 // KubeImages represents Docker images used for Kubernetes components based on Kubernetes version
 var KubeImages = map[api.OrchestratorVersion]map[string]string{
+	api.Kubernetes171: {
+		"hyperkube":    "hyperkube-amd64:v1.7.1",
+		"dashboard":    "kubernetes-dashboard-amd64:v1.6.1",
```

Comment:
This PR started out as v1.7.0, and I'd like it to continue to be focused on that. Each Kubernetes version needs to be vetted, run through our CI, Windows artifacts created, etc.